nemesis/nemesis_pathing.py

```diff
@@ -2395,24 +2395,114 @@ def gagazet(checkpoint):
         x = 0
         y = 0
     if checkpoint == 44:
-        x = 0
-        y = 0
+        x = 8
+        y = -51
     if checkpoint == 45:
-        x = 0
-        y = 0
+        x = 19
+        y = 81
     if checkpoint == 46:
-        x = 0
-        y = 0
+        x = 45
+        y = 241
     if checkpoint == 47:
-        x = 0
-        y = 0
+        x = 63
+        y = 349
     if checkpoint == 48:
+        x = 9
```

Comment:
I would suggest looking at the new hotness we are doing for pathing, but since this isn't already converted, it's not a big deal

Reply:
Yes, known concern. Nemesis is due for the same updates we did through the rest of the run. I've asked for that effort to hold off while we're still getting Nemesis working, and even made some changes recently.